probe/docker/container.go

```diff
@@ -206,6 +207,7 @@ func (c *container) GetNodeMetadata() report.NodeMetadata {
 	c.RLock()
 	defer c.RUnlock()
 
+	labels, _ := json.Marshal(c.container.Config.Labels)
```

Comment:
Never ignore errors.

(This will go away with Tom's suggestion, but just for the record...)